### PR TITLE
Add Blink oauth

### PR DESCRIPTION
**Pull request by liavt**
### What does this PR do?

Adding uninstallation instructions to the Blink integration to allow for OAuth support

### Motivation

Allow Datadog customers to have an easier way to integrate with Blink via a OAuth connection

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Also included are changes to some of the visual content to make it in-line with the marketing of the marketplace listing, and an additional codeowner from the Blink team.

### CODE REVIEW

File: blink/manifest.json [hunk 1]
```diff
@@ -6,25 +6,25 @@
   "tile": {
     "overview": "README.md#Overview",
     "configuration": "README.md#Setup",
-    "description": "No-code automation for modern CloudOps.",
+    "description": "Blink is a no-code automation platform for security and infrastructure",
```
Comment (drichards-87) [inline blink/manifest.json:9]:
```suggestion
    "description": "Blink is a no-code automation platform for security and infrastructure.",
```

File: blink/README.md [hunk 2]
```diff
@@ -2,19 +2,31 @@
...
-3. View incidents or events from the Datadog event stream in Blink
+The user interface and [automation library][2] come with over 5,000 cloud-native workflows that make it easy to create new automations. Blink helps you achieve better cloud efficiency and more competitive SLA's, with fewer operational bottlenecks.
 
-View our full list of actions and capabilities on the [documentation site][3].
+This out-of-the-box integration enables you to:
+
+- Trigger event-based Blink automations using Datadog incidents
+- Create and update Datadog incidents automatically from within Blink
+- View incidents or events from the Datadog Events Explorer in Blink
+- Automatically enrich and remediate Datadog incidents using Blink automations
```
Comment (drichards-87) [inline blink/README.md:14]:
```suggestion
- Trigger event-based Blink automations using Datadog incidents.
- Create and update Datadog incidents automatically from within Blink.
- View incidents or events from the Datadog Events Explorer in Blink.
- Automatically enrich and remediate Datadog incidents using Blink automations.
```

File: blink/README.md [hunk 3]
```diff
@@ -2,19 +2,31 @@
 
 ## Overview
 
-The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual editor to drag-and-drop Datadog API actions into your workflow or use one of the hundreds of existing pre-made automations from the [library][2].
+[Blink][1] is a low-code/no-code (LNCN) platform that enables automated incident response, cloud-native operations, and security operations workflows. Blink transforms manual tasks into interactive automations backed by the security and reliability of a cloud-native platform. Every script or ticket becomes a fully-managed automation.
```
Comment (drichards-87) [inline blink/README.md:5]:
```suggestion
[Blink][1] is a low-code/no-code (LCNC) platform that enables automated incident response, cloud-native operations, and security operations workflows. Blink transforms manual tasks into interactive automations backed by the security and reliability of a cloud-native platform. Every script or ticket becomes a fully-managed automation.
```

**Review by drichards-87 — APPROVED** [review]
Left you some minor feedback from the Docs team. PR is approved.

File: blink/README.md [hunk 4]
```diff
@@ -2,19 +2,31 @@
...
+- View incidents or events from the Datadog Events Explorer in Blink
+- Automatically enrich and remediate Datadog incidents using Blink automations
+
+For more information about Blink, see the [Blink documentation][3].
 
 ## Setup
 
 Visit [our documentation][4] for details on how to connect your Datadog workspace to Blink.
 
+### Uninstallation
```
Comment (emarsha94) [inline blink/README.md:22]:
```suggestion
## Uninstallation
```

File: blink/README.md [hunk 5]
```diff
@@ -2,19 +2,31 @@
...
+
+To uninstall the integration, simply delete the corresponding Datadog connection in your Blink workspace.
+
+Once deleted, any previous authorizations or access tokens are revoked.
+
+Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the Datadog [API Keys][5] page.
```
Comment (emarsha94) [inline blink/README.md:28]:
```suggestion
Once deleted, any previous authorizations or access tokens are revoked. Ensure that all API keys associated with this integration have been disabled by searching for the integration name on the Datadog [API Keys][5] page.
```

File: blink/README.md [hunk 6]
```diff
@@ -31,10 +43,11 @@ Blink does not include any metrics, however you can query and list metrics from
 
 ## Troubleshooting
 
-Need help? Contact [Datadog support][5].
+Need help? Contact [Datadog support][6].
```
Comment (emarsha94) [inline blink/README.md:46]: This should be Blink's support

File: blink/README.md [hunk 7]
```diff
@@ -2,19 +2,31 @@
...
 ## Overview
 
-The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual editor to drag-and-drop Datadog API actions into your workflow or use one of the hundreds of existing pre-made automations from the [library][2].
+[Blink][1] is a low-code/no-code (LNCN) platform that enables automated incident response, cloud-native operations, and security operations workflows. Blink transforms manual tasks into interactive automations backed by the security and reliability of a cloud-native platform. Every script or ticket becomes a fully-managed automation.
 
-This integration enables you to:
-1. Trigger automations within Blink in response to new Datadog incidents 
-2. Create and update Datadog incidents in Blink 
-3. View incidents or events from the Datadog event stream in Blink
+The user interface and [automation library][2] come with over 5,000 cloud-native workflows that make it easy to create new automations. Blink helps you achieve better cloud efficiency and more competitive SLA's, with fewer operational bottlenecks.
```
Comment (emarsha94) [inline blink/README.md:7]: Is this associated to Datadog? We try to avoid marketing material for integrations

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: blink/README.md (hunk 7 above)
Comment (liavt) [reply blink/README.md:7]: It isn't directly associated to Datadog. To link it more to the integration, we can possibly change it to be a link to premade Datadog automations to show examples of how to use the two platforms.

**Review by emarsha94 — APPROVED** [review]

**Review by nmuesch — APPROVED** [review]

File: blink/README.md [hunk 8]
```diff
@@ -2,19 +2,29 @@
...
+- Trigger event-based Blink automations using Datadog incidents.
+- Create and update Datadog incidents automatically from within Blink.
+- View incidents or events from the Datadog Events Explorer in Blink.
+- Automatically enrich and remediate Datadog incidents using Blink automations.
+
+For more information about Blink, see the [Blink documentation][3].
 
 ## Setup
 
 Visit [our documentation][4] for details on how to connect your Datadog workspace to Blink.
```
Comment (eho1307) [inline blink/README.md:20]: Hi @liavt, apologies for the delay. Due to a recent incident, we were in a code freeze for the last two weeks. Could you confirm this document will be updated with instructions specific to OAuth?

Comment (liavt) [reply blink/README.md:20]: @eho1307 Yes, once the Oauth is deployed we will update the documentation link with instructions on how to connect

**Comment by eho1307** [comment]
@liavt Let us know once you've confirmed OAuth authorization works from a different Datadog site. Once that's complete, we can go ahead and merge this PR.

**Comment by emarsha94** [comment]
> @liavt Let us know once you've confirmed OAuth authorization works from a different Datadog site. Once that's complete, we can go ahead and merge this PR.

Hi @liavt wanted to follow up on this and see if you were able to test the above. Thanks!

**Pull request merged**